app-frontend/src/app/pages/projects/edit/browse/browse.controller.js

```diff
@@ -285,14 +288,23 @@ export default class ProjectAddScenesBrowseController {
     }
 
     onQueryParamsChange() {
-        const filterObject = Object.assign(this.queryParams, { forProjectId: this.project.id });
-        this.sessionStorage.set('filters', filterObject);
-        this.$state.go('.', this.getCombinedParams(), {
-            notify: false,
-            inherit: false,
-            location: 'replace'
+        const filterObject = Object.assign(this.queryParams, {
+            forProjectId: this.project.id,
+            dataRepo: this.sourceRepo
         });
-        this.requestNewSceneList();
+        this.sessionStorage.set('filters', filterObject);
```

Comment:
It might make sense to namespace browse filters to projects, instead of the session in general - if a user is working on two projects in different countries, they probably only care about the country that the project they are currently working on is in.
You could do that with something like ```this.sessionStorage.set(`${projectid}-filters`, filterObject);```, and if a project doesn't have set filters, keep track of the last project they were in and duplicate the filters

Reply:
on second thought, we should probably just make an issue to discuss this w/ designers

Reply:
Agreed. This session storage will actually be removed in the next commit I am about to make. @Lknechtli